Dockerfile

```diff
@@ -33,6 +33,8 @@ RUN apt-get update && apt-get install -y libatlas-base-dev libopenblas-dev libop
 RUN apt-get install -y libzmq3-dev python-pip default-jdk && \
     apt-get install -y python-dev libcurl4-openssl-dev && \
     pip install jupyter pycurl && \
+    # Install older tornado - https://github.com/jupyter/notebook/issues/4437
+    pip install "tornado<6" && \
```

Comment:
<6 is fine given the tests are passing.

Just a FYI, we also have a similar pin for tornado on the Docker Python image:

https://github.com/Kaggle/docker-python/blob/master/Dockerfile#L351